aat/src/aat/resources/features/F-093/S-600.2.td.json

```diff
@@ -25,7 +25,7 @@
       "Content-Length": "[[ANYTHING_PRESENT]]"
     },
     "body": {
-      "__plainTextValue__" : "'retainHiddenValue' has been incorrectly configured or is invalid for fieldID ['CollectionComplexField'] on ['CaseEventToFields']"
+      "__plainTextValue__" : "'retainHiddenValue' has been incorrectly configured or is invalid for fieldID ['CollectionComplexField'] on ['CaseEventToFields']'retainHiddenValue' has been incorrectly configured or is invalid for fieldID ['CollectionComplexField'] on ['CaseEventToFields']'retainHiddenValue' has been incorrectly configured or is invalid for fieldID ['CollectionComplexField'] on ['CaseEventToFields']'retainHiddenValue' has been incorrectly configured or is invalid for fieldID ['CollectionComplexField'] on ['CaseEventToFields']"
```

Comment:
Could this please be at least separated by a delimiter like , or \n for better readability?

Reply:
I mean, in all similar places.

Reply:
In the api response, I am separating lines with \n delimiter. But,  **__plainTextValue__** is populated after \n delimiter is removed from the result in BeftaFw, and the result comes as I have put in the test case. Please check it out below line in BeftaFw;

https://github.com/hmcts/befta-fw/blob/e815128ada4c703d10f9c9c8e85629481abaca8f/src/main/java/uk/gov/hmcts/befta/player/DefaultBackEndFunctionalTestScenarioPlayer.java#L468